### PR TITLE
Rename /apis to /api

### PR DESCRIPTION
**Pull request by Jaanki**
and make relevant necessary changes

Partially-Closes: #1329
Signed-Off-By: Janki Chhatbar <jchhatba@redhat.com>

### CODE REVIEW

File: config/broker/broker-admin/role.yaml [hunk 1]
```diff
@@ -52,9 +52,6 @@ rules:
       - delete
   - apiGroups:
       - discovery.k8s.io
-    resources:
-      - endpointslices
-      - endpointslices/restricted
```
Comment (tpantelis) [inline config/broker/broker-admin/role.yaml]: Why was this change made?

File: config/manager/manager.yaml [hunk 2]
```diff
@@ -14,9 +14,6 @@ metadata:
   labels:
     control-plane: submariner-operator
 spec:
-  selector:
-    matchLabels:
-      control-plane: submariner-operator
```
Comment (tpantelis) [inline config/manager/manager.yaml]: ...and this

File: config/rbac/networkplugin_syncer/cluster_role.yaml [hunk 3]
```diff
@@ -4,27 +4,3 @@ kind: ClusterRole
...
-      - list
-      - watch
-  - apiGroups:
-      - submariner.io
-    resources:
-      - endpoints
-    verbs:
-      - get
-      - list
-      - watch
```
Comment (tpantelis) [inline config/rbac/networkplugin_syncer/cluster_role.yaml]: ...and this

File: config/broker/broker-admin/role.yaml (hunk 1 above)
Comment (Jaanki) [reply config/broker/broker-admin/role.yaml]: seems like side-effect of a rebase....Will check

**Review by tpantelis — APPROVED** [review]

**Review by skitt — APPROVED** [review]

**Pull request merged**